pkg/roachpb/span_config.proto

```diff
@@ -151,6 +166,36 @@ message SpanConfigEntry {
   SpanConfig config = 2 [(gogoproto.nullable) = false];
 };
 
+// SystemSpanConfig is a system installed configuration that may apply to
+// multiple spans.
+message SystemSpanConfig {
+  option (gogoproto.equal) = true;
+
+  // ProtectionPolicies is a list of policies which protect data from being
+  // GC-ed.
+  repeated ProtectionPolicy protection_policies = 1 [(gogoproto.nullable) = false];
+}
+
+// SystemSpanConfigTarget is used to specify the target of a SystemSpanConfig.
+message SystemSpanConfigTarget {
+   // TenantID indicates the tenant ID of the logical cluster being targeted.
+   // For secondary tenants this field is left unset. For the host we can use
+   // this field to protect a specific secondary tenant.
```

Comment:
[minor] Maybe a TODO to later assert that this is the case when wiring things up for real.

Reply:
Done.